sql/2022/12/opportunity-score-wp-sites-lcp-without-fetchpriority-high.sql

```diff
@@ -0,0 +1,96 @@
+# HTTP Archive query to get Performance Lab plugin version distribution in a given month.
```

Comment:
@kt-12 the comment needs to update. Something like below. Please update it as per your result.

```suggestion
# HTTP Archive query to get usage of LCP image measurements in a given month.
```